packages/Chem/src/widgets/scaffold-tree.ts

```diff
@@ -1136,9 +1154,21 @@ export class ScaffoldTreeViewer extends DG.JsViewer {
       this.updateFilters();
       this._bitOpInput!.value = this.bitOperation;
     }
+    else if (p.name === 'size') {
```

Comment:
Why would be reconstruct the whole tree when the size is changed?

Reply:
Fixed